src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1121,13 +1132,30 @@ class AbstractUtxoCoin extends BaseCoin {
       const baseKeyPath = deriveKeys(deriveKeys(keys, 0), 0);
       const userKeyArray = deriveKeys(baseKeyPath, 0);
       const changeKeyArray = deriveKeys(baseKeyPath, 1);
+      const userKeyArraySW = deriveKeys(baseKeyPath, 10);
+      const changeKeyArraySW = deriveKeys(baseKeyPath, 11);
 
       // Collect the unspents
       const addressesById = {};
-      const userUnspents = yield queryBlockchainUnspentsPath(userKeyArray, '/0/0/0');
-      const changeUnspents = yield queryBlockchainUnspentsPath(changeKeyArray, '/0/0/1');
-      const unspents = userUnspents.concat(changeUnspents);
-
+      // Gather the queries, to be executed in parallel
+      const includeP2SH_P2WSH = _.isUndefined(params.ignoreAddressTypes) || _.isEmpty(params.ignoreAddressTypes) || !_.includes(params.ignoreAddressTypes, AbstractUtxoCoin.AddressTypes.P2SH_P2WSH);
+      let queries = [
+        queryBlockchainUnspentsPath(userKeyArray, '/0/0/0', false),
+        queryBlockchainUnspentsPath(changeKeyArray, '/0/0/1', false)]
```

Comment:
Missing semi colon here, and the closing array bracket should be on it's own line.